docs/docs/intro.md

```diff
@@ -316,6 +317,33 @@ Then rerun the GraphQL query again and you should get a response similar to:
 }
 ```
 
+## Query via GraphQL
+
+The generated GraphQL schema allows you to access any node in your entity graph via the following Query:
+
+```graphql title="src/graphql/schema.gql
+type Query {
+  node(id: ID!): Node
+}
+```
+
+`Node` is a polymorphic type and can accept an ID for any type of object. You can query for the User you just created with the following GraphQL query - replacing the ID with the ID returned when you created the user!
+
+```graphql
+query ByID {
+  node(id:"bm9kZTp1c2VyOjQ1Y2RkNmUyLWY2ZmItNDVlMC1iNWIwLWEwN2JlZWVmM2QxOQ==") {
+    id
+    ... on User {
+      firstName
+      lastName
+      emailAddress
+    }
+  }
+}
+```
+
+Note: The ID exposed to GraphQL is not the same as the ID for the row in the database. GraphQL IDs should be considered opaque by clients of your service.
```

Comment:
yah, i need to add a whole section about GraphQL IDs, Connection etc linking to the relevant relay docs.

I think this is fine but is it worth linking to https://relay.dev/docs/guides/graphql-server-specification/#object-identification or https://graphql.org/learn/global-object-identification/

Neither link really explains what's happening here though.

Reply:
That second link looks good. I'll update to include a link there as well.